flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-starrocks/src/main/java/com/ververica/cdc/connectors/starrocks/sink/StarRocksUtils.java

```diff
@@ -0,0 +1,341 @@
+     * @param zoneId the time zone used when converting from <code>TIMESTAMP WITH LOCAL TIME ZONE
+     *     </code>
+     */
+    public static RecordData.FieldGetter createFieldGetter(
+            DataType fieldType, int fieldPos, ZoneId zoneId) {
+        final RecordData.FieldGetter fieldGetter;
+        // ordered by type root definition
+        switch (fieldType.getTypeRoot()) {
+            case BOOLEAN:
+                fieldGetter = record -> record.getBoolean(fieldPos);
+                break;
+            case TINYINT:
+                fieldGetter = record -> record.getByte(fieldPos);
+                break;
+            case SMALLINT:
+                fieldGetter = record -> record.getShort(fieldPos);
+                break;
+            case INTEGER:
+                fieldGetter = record -> record.getInt(fieldPos);
+                break;
+            case BIGINT:
+                fieldGetter = record -> record.getLong(fieldPos);
+                break;
+            case FLOAT:
+                fieldGetter = record -> record.getFloat(fieldPos);
+                break;
+            case DOUBLE:
+                fieldGetter = record -> record.getDouble(fieldPos);
+                break;
+            case DECIMAL:
+                final int decimalPrecision = getPrecision(fieldType);
+                final int decimalScale = getScale(fieldType);
+                fieldGetter =
+                        record ->
+                                record.getDecimal(fieldPos, decimalPrecision, decimalScale)
+                                        .toBigDecimal();
+                break;
+            case CHAR:
+            case VARCHAR:
+                fieldGetter = record -> record.getString(fieldPos).toString();
+                break;
+            case DATE:
+                fieldGetter =
+                        record ->
+                                DATE_FORMATTER.format(
+                                        Date.valueOf(
+                                                LocalDate.ofEpochDay(record.getInt(fieldPos))));
+                break;
+            case TIMESTAMP_WITHOUT_TIME_ZONE:
+                fieldGetter =
+                        record ->
+                                record.getTimestamp(fieldPos, getPrecision(fieldType))
+                                        .toLocalDateTime()
+                                        .format(DATETIME_FORMATTER);
+                break;
+            case TIMESTAMP_WITH_LOCAL_TIME_ZONE:
+                fieldGetter =
+                        record ->
+                                ZonedDateTime.ofInstant(
+                                                record.getLocalZonedTimestampData(
+                                                                fieldPos, getPrecision(fieldType))
+                                                        .toInstant(),
+                                                zoneId)
```

Comment:
add `.withZoneSameInstant(ZoneId.systemDefault())` to change Time Zone Information.